.github/workflows/issues-to-projects.yml

```diff
@@ -7,22 +8,42 @@
 # - read:org
 #
 
-name: Add issues to Organization Project Boards
+name: Add issues and PRs to Organization Project Boards
 
 on:
   issues:
     types: [ opened, edited, reopened, labeled ]
+  pull_request_target:
+    types: [ review_requested ]
+
+# map fields with customized labels
+env:
+  pending_review: 👀 Pending review
 
 jobs:
-  insert:
+  assign_boost_issues:
+    name: Assign Boost issues to Boost board.
     runs-on: ubuntu-latest
-
+    if: github.event_name == 'issues' && (github.event.action == 'opened' || github.event.action == 'edited' || github.event.action == 'reopened' || github.event.action == 'labeled' )
```

Comment:
I think checking `github.event.action` is unnecessary, because the job should not run for other action types then these.

Reply:
You're right. I removed the check in fef698749b0f9f676bec7cc47fcaeb627cbf2ab7